sensniff.py

```diff
@@ -330,22 +368,40 @@ def handle(self, data):
             self.__open_fifo()
 
         if self.of is not None:
-            try:
-                if self.needs_pcap_hdr is True:
-                    self.of.write(pcap_global_hdr)
-                    self.needs_pcap_hdr = False
-                self.of.write(data.pcap)
-                self.of.flush()
-                logger.info('Wrote a frame of size %d bytes' % (data.len))
-                stats['Piped'] += 1
-            except IOError as e:
-                if e.errno == errno.EPIPE:
-                    logger.info('Remote end stopped reading')
-                    stats['Not Piped'] += 1
-                    self.of = None
-                    self.needs_pcap_hdr = True
-                else:
-                    raise
+            if platform.system() == 'Windows':
+				try:
+					if self.needs_pcap_hdr is True:
+						win32file.WriteFile(self.of, pcap_global_hdr) #self.of.write(pcap_global_hdr)
+						self.needs_pcap_hdr = False
+					win32file.WriteFile(self.of, data.pcap) #self.of.write(data.pcap)
+					#self.of.flush()
+					logger.info('Wrote a frame of size %d bytes' % (data.len))
+					stats['Piped'] += 1
+				except IOError as e:
+					if e.errno == errno.EPIPE:
+						logger.info('Remote end stopped reading')
+						stats['Not Piped'] += 1
+						self.of = None
+						self.needs_pcap_hdr = True
+					else:
+						raise
+            else:
+				try:
+					if self.needs_pcap_hdr is True:
+						self.of.write(pcap_global_hdr)
+						self.needs_pcap_hdr = False
+					self.of.write(data.pcap)
+					self.of.flush()
+					logger.info('Wrote a frame of size %d bytes' % (data.len))
+					stats['Piped'] += 1
+				except IOError as e:
+					if e.errno == errno.EPIPE:
+						logger.info('Remote end stopped reading')
+						stats['Not Piped'] += 1
+						self.of = None
+						self.needs_pcap_hdr = True
+					else:
+						raise
```

Comment:
I agree. Actually, this first PR aims at providing a basic 'just working' script on Windows. I am totally beginner with Python. Maybe someone more comfortable with Python may re-implement this section in order to make things look beautiful. What I propose is to close this PR if you expect a better implementation, and I will keep things 'as is' on my fork.